javalin/src/main/java/io/javalin/plugin/metrics/MicrometerPlugin.kt

```diff
@@ -49,8 +51,12 @@ class MicrometerPlugin @JvmOverloads constructor(private val registry: MeterRegi
                             .findAny()
                             .map(HandlerEntry::path)
                             .map { path: String -> if (path == "/" || StringUtils.isBlank(path)) "root" else path }
-                            .map { path: String -> if (response.status in 300..399) "REDIRECTION" else path }
-                            .map { path: String -> if (response.status == 404) "NOT_FOUND" else path }
+                            .map { path: String ->
+                                if (!tagRedirectResponses && response.status in 300..399) "REDIRECTION" else path
```

Comment:
This looks backwards?

```kotlin
val dontTagRedirects = !tagRedirectResponses
val isRedirect= response.status in 300..399
if (dontTagRedirects && isRedirect) "REDIRECTION" else path
```

Reply:
My variable name is bad :) - Would `tagRedirectPaths` and `tagNotFoundMappedPaths` be better?

Basically, the behavior that I want to introduce is to tag redirect responses with their individual path, instead of the current case, that tags them all together with a `REDIRECTION` value.